sdk/template/Azure.Template/stress/Azure.Template.Stress.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I wonder if it's too early to add this project to the template. The template is used by onboarding teams to start a library and I doubt any of them needs a stress project just yet.

Reply:
I do want a template showing how to add stress tests though.  Where else could it live?  And the onboarding training does include perf and stress, so I think it's good for teams to at least think about right away.